src/main/java/org/tugraz/sysds/runtime/instructions/cp/UnaryFrameCPInstruction.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The license is wrong, please use your TU Graz copyright as we do for other new files.